src/trello/views/task_view.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from rest_framework import generics
+from rest_framework.generics import get_object_or_404
+from trello.permissions import *
+from trello.models import *
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from trello.serializers.tasks_serializers import *
+
+
+class TaskCreateView(generics.CreateAPIView):
+	serializer_class = TaskDetailSerializer
+	permission_classes = (IsAuthenticated, )
+
+	def perform_create(self, serializer):
+		column = Column.objects.get(id=self.request.data.get('column'))
+		serializer.save(owner=self.request.user)
+		serializer.save(column=column)
+
+
+class TaskDetailView(generics.RetrieveUpdateDestroyAPIView):
+	serializer_class = TaskDetailSerializer
+	queryset = Task.objects.all()
+	permission_classes = (IsAuthenticated, IsOwnerReadOnly, IsAdminUser)
```

Comment:
Access to projects and all objects in project can only have owner of this projects and admins of this project(not django admin)